homework_3.py

```diff
@@ -0,0 +1,43 @@
+from flask import Flask
+from sqlalchemy import SQLAlchemy
+from datetime import datetime
+from pydantic import BaseModel
+from datetime import date
+
+app = Flask(__name__)
+
+
+@app.route('/health')
+def health_check():
+    return '', 200
+
+
+db = SQLAlchemy()
+
+
+class AnimalModel(BaseModel):
+    name: str
+    breed: str
+    birthdate: date
+    photo_url: str
+    age: int
+
+
+class Animal(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String(120), nullable=False)
+    breed = db.Column(db.String(120), nullable=False)
+    birthdate = db.Column(db.Date, nullable=False)
+    photo_url = db.Column(db.String(255), nullable=False)
+
+
+def calculate_age(birthdate):
```

Comment:
You can move this logic to your pydantic model as computed field
https://github.com/marcustas/course_flask/pull/9/commits